src/components/Modals/NewSemester.vue

```diff
@@ -105,19 +105,25 @@
   </div>
 </template>
 
-<script>
+<script lang="ts">
+import Vue, { PropType } from 'vue';
 import { clickOutside } from '@/utilities';
 
 import fall from '@/assets/images/fallEmoji.svg';
 import spring from '@/assets/images/springEmoji.svg';
 import winter from '@/assets/images/winterEmoji.svg';
 import summer from '@/assets/images/summerEmoji.svg';
 import {
+  // @ts-ignore
```

Comment:
Why do these need to be ts-ignored?

Reply:
Tried to add stub types for scss but this destructuring syntax just doesn't work :(